From 890af36a3939f0978af779caa878ddeb44183d8d Mon Sep 17 00:00:00 2001
From: Andrea Mazzoleni <amadvance@gmail.com>
Date: Tue, 8 May 2018 19:32:44 +0200
Subject: [PATCH 3/3] Remove the --enable-static configure option

I doubt it still work and it doesn't make sense anymore.
---
 configure.ac | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4e609993..1c94e037 100644
--- a/configure.ac
+++ b/configure.ac
@@ -139,13 +139,6 @@ AC_ARG_ENABLE(
 )
 AC_SUBST([CONF_DEBUG],[$ac_enable_debug])
 
-AC_ARG_ENABLE(
-	[static],
-	AC_HELP_STRING([--enable-static],[enable static compilation. (default no)]),
-	[ac_enable_static=$enableval],
-	[ac_enable_static=no]
-)
-
 AC_ARG_ENABLE(
 	[bare],
 	AC_HELP_STRING([--enable-bare],[enable compilation without drivers. (default no)]),
@@ -232,9 +225,6 @@ if test $ac_auto_ldflags = yes ; then
 	else
 		LDFLAGS="-s"
 	fi
-	if test $ac_enable_static = yes ; then
-		LDFLAGS="-static $LDFLAGS"
-	fi
 	if test ! -z $ac_host_ldflags; then
 		LDFLAGS="$ac_host_ldflags $LDFLAGS"
 	fi
-- 
2.17.0

